### PR TITLE
chrome-stable: fix port 9222 not being exposed (using a socat tunnel)

### PR DESCRIPTION
**Pull request by DoLooP**

### CODE REVIEW

**Comment by DoLooP** [comment]
I'm using this image in docker compose as multiple replicas.
i can't expose ports using docker-compose.yml `ports: - "127.0.0.1:9222:9222"` because the replicas are conflicting on host port 9222.

File: chrome-stable/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#! bin/sh
+socat tcp-listen:9222,reuseaddr,fork tcp:localhost:9223 &
+/usr/bin/google-chrome --disable-gpu --headless --disable-dev-shm-usage --remote-debugging-port=9223 --user-data-dir=/data
```
Comment (yukinying) [inline chrome-stable/entrypoint.sh:3]: Can we make this script support taking additional arguments, i.e. adding $@ to the end of this line?

**Review by yukinying — COMMENTED** [review]
Thank you. That's great. I have put a comment in the entrypoint.sh, other than that everything LGTM.

File: chrome-stable/entrypoint.sh (hunk 1 above)
Comment (yukinying) [reply chrome-stable/entrypoint.sh:3]: I will make another PR.

**Pull request merged**